src/Facebook/InstantArticles/Elements/InstantArticle.php

```diff
@@ -396,6 +396,7 @@ public function addMetaProperty($property_name, $property_content)
 
     public function render($doctype = '<!doctype html>', $format = false)
     {
+        $doctype = is_null($doctype) ? '<!doctype html>' : $doctype;
```

Comment:
My understanding is that even with the signature change, if somebody is using this method as either `$article->render()` or `$article->render(null)`, the `$doctype` will be set to `<!doctype html>` because of the line you added. Maybe I am missing something?